src/main/java/io/boomerang/service/crud/TeamServiceImpl.java

```diff
@@ -273,19 +279,45 @@ public List<FlowTeamConfiguration> getAllTeamProperties(String teamId) {
 
     if (flowTeamEntity.getSettings() != null
         && flowTeamEntity.getSettings().getProperties() != null) {
+    	filterPasswordValue(flowTeamEntity.getSettings() == null ? null:flowTeamEntity.getSettings().getProperties(), true);
       return flowTeamEntity.getSettings().getProperties();
     } else {
       return Collections.emptyList();
     }
   }
 
+	private List<? extends AbstractConfigurationProperty> filterPasswordValue(
+			List<? extends AbstractConfigurationProperty> properties, boolean isDefaultValue) {
+		final String passTypeStr = "password";
+		// If the property is a password, do not return its value, for security reasons.
+		if (properties == null)
+			return null;
+		Optional<? extends AbstractConfigurationProperty> passProp = properties.stream()
+				.filter(f -> passTypeStr.equals(f.getType())
+						&& (isDefaultValue ? f.getDefaultValue() != null : f.getValue() != null))
+				.findAny();
+		if (passProp.isPresent()) {
+			if (isDefaultValue) {
+				passProp.get().setDefaultValue(null);
+			} else {
+				passProp.get().setValue(null);
+			}
+		}
+		return properties;
+	}
+  
+  
```

Comment:
Can we make this a common util method in `io.boomerang.util` package that can be used by any impl over time? and make it so both team and workflow can use the same shared methods (even if there are two or three) it centralizes the filtering code.